src/types/rawJSON.ts

```diff
@@ -0,0 +1,45 @@
+export default interface rawJSON {
```

Comment:
Ah ignore my earlier comment, you've done that here!

Ace.
Feel free to keep this but you can also cut this down if there are fields here you _know_ you won't use.

Typescript only typechecks data you extract and use within your app